charts/kubernetes-agent/values.yaml

```
@@ -331,6 +331,12 @@ persistence:
         pullPolicy: IfNotPresent
         tag: "0.2.0"
 
+upgrade:
+  username: ""
+  password: ""
+  registry: ""
+    
+
```

```suggestion
upgrade:
  dockerAuth:
    username: ""
    password: ""
    registry: ""
```
Consider changing the root from `upgrade` - it's not very descriptive.